### PR TITLE
Send routine to patient

### PR DESCRIPTION
**Pull request by AlexLeathers**
Can now create and send routines to the patient.

### CODE REVIEW

File: PhysicalTherapy/PhyscialTherapy/Client/src/app/therapist-routine-creation-screen/therapist-routine-creation-screen.component.html [hunk 1]
```diff
@@ -20,15 +20,15 @@
...
+          <input id="target{{i}}" formControlName="targetIndex" placeholder="Change positions?" type="number">
+          <input id="exName{{i}}" formControlName="exerciseName" placeholder="NAME" list="exerciseList">
         </div>
         <div>
-          <input formControlName="holdLength" placeholder="Hold Length">
-          <input formControlName="reps" placeholder="Number of reps">
-          <input formControlName="sets" placeholder="Number of sets">
+          <input id="hold{{i}}" formControlName="holdLength" placeholder="Hold Length">
+          <input id="reps{{i}}" formControlName="reps" placeholder="Number of reps">
+          <input id="sets{{i}}" formControlName="sets" placeholder="Number of sets">
```
Comment (jtzhugh) [inline PhysicalTherapy/PhyscialTherapy/Client/src/app/therapist-routine-creation-screen/therapist-routine-creation-screen.component.html]: Why did you add the ID field here instead of accessing these through the form array?

Comment (AlexLeathers) [reply PhysicalTherapy/PhyscialTherapy/Client/src/app/therapist-routine-creation-screen/therapist-routine-creation-screen.component.html]: Just didn't notice those at first. :P I'll change it.

Comment (jtzhugh) [reply PhysicalTherapy/PhyscialTherapy/Client/src/app/therapist-routine-creation-screen/therapist-routine-creation-screen.component.html]: Alright! You can leave them in too. I was just worried that it didn't work as well as I thought. :P

Comment (AlexLeathers) [reply PhysicalTherapy/PhyscialTherapy/Client/src/app/therapist-routine-creation-screen/therapist-routine-creation-screen.component.html]: I changed it and it still works. :)

File: PhysicalTherapy/PhyscialTherapy/Controllers/RoutineExerciseController.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        public RoutineExerciseController(IRoutineExerciseRepository routineExerciseRepository)
+        {
+            _routineExerciseRepository = routineExerciseRepository;
+        }
+
+        [HttpGet]
+        [Produces(typeof(DbSet<RoutineExercise>))]
+        public IActionResult GetRoutineExercises()
+        {
+            var test = new ObjectResult(_routineExerciseRepository.GetAll());
```
Comment (hungaud) [inline PhysicalTherapy/PhyscialTherapy/Controllers/RoutineExerciseController.cs]: me doing var test was just me debuggin the object to be return. dont really need this part.

**Comment by hungaud** [comment]
looks good to me. would probably have to messa roudn with this just to really understand, but at first glace everything looks good

File: PhysicalTherapy/PhyscialTherapy/Controllers/RoutineExerciseController.cs (hunk 2 above)
Comment (AlexLeathers) [reply PhysicalTherapy/PhyscialTherapy/Controllers/RoutineExerciseController.cs]: I'll just merge and remove on one of my next branches.
